addon/i3dio/node_classes/material.py

```diff
@@ -144,9 +163,16 @@ def _emissive_from_nodes(self, node):
                     return
             self.logger.debug("Has no Emissivemap")
         r, g, b, a = emission_c
-        if (0, 0, 0, 1) != (r, g, b, a):
-            self.logger.debug("Write emissiveColor")
-            self._write_emission(emission_c)
+
+        if blender_version >= 4:

```

Comment:
```suggestion
        if bpy.app.version >= (4, 0, 0)
```
I believe this would be better since the other function you made uses bpy anyway and Blender version is always a 3-value tuple

Reply:
This would check just for 4.0 not for 4.1 and newer versions

Reply:
It is doing "Bigger than" and tuples are compared lexographically element-by-element.

```python
version = (4, 5, 2)

if version >= (4, 0, 0):
    print(f"{version} is higher/equal than/to 4.0.0")
if version >= (4, 3, 1):
    print(f"{version} is higher/equal than/to 4.3.1")
if version >= (4, 5, 1):
    print(f"{version} is higher/equal than/to 4.5.1")
if not version >= (4, 7, 2):
    print(f"{version} is lower/equal than/to 4.7.2")
```

```shell
(4, 5, 2) is higher/equal than/to 4.0.0
(4, 5, 2) is higher/equal than/to 4.3.1
(4, 5, 2) is higher/equal than/to 4.5.1
(4, 5, 2) is lower/equal than/to 4.7.2 
```

Reply:
Yes, I forgot about that, in that case, it can be implemented as u have written